provision/aws/rosa_create_cluster.sh

```diff
@@ -15,6 +15,7 @@ if [ -z "$REGION" ]; then echo "Variable REGION needs to be set."; exit 1; fi
 if [ -z "$COMPUTE_MACHINE_TYPE" ]; then echo "Variable COMPUTE_MACHINE_TYPE needs to be set."; exit 1; fi
 
 if [ "$MULTI_AZ" = "true" ]; then MULTI_AZ_PARAM="--multi-az"; else MULTI_AZ_PARAM=""; fi
```

Comment:
any reason why the AZ logic is not under this `if` block instead, as we are defining multiple AZ's for the cluster using MULTI_AZ set to true ?

Reply:
Chasing PRs: @kami619 / @mhajas - has this comment been resolved? Can it be merged?

Reply:
yup @ahus1 I misunderstood the multi-az definition and Michal clarified it offline. The PR looks good to me to be merged.